packages/remix-google-cloud-functions/__tests__/server-test.ts

```diff
@@ -0,0 +1,271 @@
+import supertest from "supertest";
+import { createRequest } from "node-mocks-http";
+import {
+  createRequestHandler as createRemixRequestHandler,
+  Response as NodeResponse,
+} from "@remix-run/node";
+import { Readable } from "stream";
```

Comment:
This one as well

Reply:
I'm not sure what you mean here? We've added `"@remix-run/node"` to devDependencies, although it's already a regular dependency. `stream` is part of node core library and it doesn't make sense to add it to dependencies